packages/fiori/src/NotificationListGroupItem.hbs

```diff
@@ -82,6 +82,9 @@
 	</ui5-list>
 
 	{{#if busy}}
-		<ui5-busy-indicator active size="Medium" class="ui5-nli-busy"></ui5-busy-indicator>
+		<ui5-busy-indicator
+				busyDelay="{{delay}}"
+				active size="Medium"
+				class="ui5-nli-busy"></ui5-busy-indicator>
```

Comment:
Please put the every attribute on new line & the closing tag on new line as well